go.mod

```diff
@@ -1,6 +1,6 @@
 go 1.22.0
 
-toolchain go1.22.2
+toolchain go1.22.6
```

Comment:
This is now how one would upgrade Go version. The above `go` directive would be what you want to change, see:

- [go.mod file reference](https://go.dev/doc/modules/gomod-ref)